broker/src/test/java/io/camunda/zeebe/broker/transport/commandapi/QueryApiRequestHandlerTest.java

```diff
@@ -312,9 +313,7 @@ void internalError() {
         .extracting(Either::getLeft)
         .extracting(
             ErrorResponse::getErrorCode, error -> BufferUtil.bufferAsString(error.getErrorData()))
-        .containsExactly(
-            ErrorCode.INTERNAL_ERROR,
-            "Failed to handle query due to internal error; see the broker logs for more");
+        .contains(ErrorCode.MALFORMED_REQUEST);
```

Comment:
I guess this could be considered a breaking API change? I'm not sure.

Reply:
Yes. This will be a breaking change. Since this is not essential for this PR, please revert this change. You can bring this up later and discuss how we can introduce this without breaking API.

Reply:
Okay, I'll change it. It's not quite so easy because that is behavior that is different between the command api and the query api. I had hoped that we don't guarantee stability for experimental APIs :sweat_smile:

Reply:
Ah. Sorry. I misunderstood it. I thought it was changed for CommanApi. If it is changed only for QueryApi, it might be ok.